### PR TITLE
Feature/aqi 4279

### PR DESCRIPTION
**Pull request by KentMiller**

### CODE REVIEW

**Comment by KentMiller** [comment]
I will have to change the version before merging this PR. It's currently a pre-release that I used for local testing.

**Review by AQI-DanG — APPROVED** [review]

File: src/Aquarius.ONE.ClientSDK/Aquarius.ONE.ClientSDK.csproj [hunk 1]
```diff
@@ -3,14 +3,14 @@
...
-	  <VersionPrefix>16.0.1</VersionPrefix>
+	  <VersionPrefix>16.1.0</VersionPrefix>
 	  <!--For Pre-release packages update the VersionSuffix -->
 	  <VersionSuffix></VersionSuffix>
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
     <Authors>Aquatic Informatics</Authors>
     <Company>Aquatic Informatics</Company>
     <Description>Aquatic Informatics Aquarius Client SDK for the Standard 2.0 Framework.</Description>
-    <Copyright>Copyright © 2022, Aquatic Informatics</Copyright>
+    <Copyright>Copyright © 2023, Aquatic Informatics</Copyright>
```
Comment (JonathanRobertson-AQI) [inline src/Aquarius.ONE.ClientSDK/Aquarius.ONE.ClientSDK.csproj:13]: thanks for changing the year here

File: src/Aquarius.ONE.ClientSDK/Common/Activity/ActivityApi.cs [hunk 2]
```diff
@@ -166,7 +168,38 @@ public async Task<bool> UpdateActivitiesAsync(Proto.Activities activities)
                 Event(e, CreateLoggerArgs(EnumLogLevel.Error, $"UpdateActivitiesAsync Failed - {e.Message}"));
                 if (_throwAPIErrors) 
 					 throw; 
-				 return false;
+				return false;
+            }
+        }
+
+        public async Task<bool> UpdateActivityPropertyBagAsync(Guid activityId, OneJsonPatchItems propertyBagUpdates)
```
Comment (JonathanRobertson-AQI) [inline src/Aquarius.ONE.ClientSDK/Common/Activity/ActivityApi.cs:175]: Can you please add a ///summary to public methods?  I think it's visible by and helpful to the API consumers.

**Review by JonathanRobertson-AQI — CHANGES_REQUESTED** [review]
requesting summary on public methods

File: src/Aquarius.ONE.ClientSDK/Common/Activity/ActivityApi.cs (hunk 2 above)
Comment (KentMiller) [reply src/Aquarius.ONE.ClientSDK/Common/Activity/ActivityApi.cs:211]: Added comments.

Comment (JonathanRobertson-AQI) [reply src/Aquarius.ONE.ClientSDK/Common/Activity/ActivityApi.cs:211]: thanks!

**Review by JonathanRobertson-AQI — APPROVED** [review]

**Pull request merged**